src/renderers/WebGLRenderer.js

```diff
@@ -653,24 +658,24 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	var deallocateTexture = function ( texture ) {
 
-		if ( texture.image && texture.image.__webglTextureCube ) {
+		if ( texture.image && objectRendererWebGLProps[texture.uuid].__image__webglTextureCube ) {
```

Comment:
> It would looks something like"

> ```javascript
>
> function getWebGLProps (obj) {
>    if ( ! objectRendererWebGLProps[object.uuid]) {
>     
>        objectRendererWebGLProps[object.uuid] = {};
>
>   }
>
>    return objectRendererWebGLProps[object.uuid];
> }

> getWebGLProps(texture)
```

I actually prefer that design. It aligns more with how I've been doing [WebGLObjects](https://github.com/mrdoob/three.js/blob/dev/src/renderers/webgl/WebGLObjects.js), [WebGLGeometries](https://github.com/mrdoob/three.js/blob/dev/src/renderers/webgl/WebGLGeometries.js) and so on.

Maybe you could move all this into a new class named `WebGLProps`?